index.js

```diff
@@ -0,0 +1,152 @@
+const express = require('express');
+const knex = require('knex');
+const knexConfig = require('./knexfile');
+const db = knex(knexConfig.development);
+const server = express();
+server.use(express.json());
+
+// PROJECTS
+
+//GET
+
+server.get("/projects", (req, res) => {
+    db("projects")
+      .then(projects => {
+          res.status(200).json(projects)
+      })
+      .catch(err => {
+          res.status(500).json(err)
+      });
+  });
+
+  server.get("/projects/:id", (req, res) => {
+    const { id } = req.params;
+    db("projects")
+      .where({ id: id})
+      .then(projects => {
+          res.status(200).json(projects)
+      })
+      .catch(err => {
+          res.status(500).json(err)
+      });
+  });
+
+
+ // POST
+
+ server.post("/projects", (req, res) => {
+    const projects = req.body;
+
+     db("projects")
+      .insert(projects)
+      .into("projects")
```

Comment:
You don't need the `into` here since you already are specifying the table two lines above it.